guides/android-setup.md

```diff
@@ -48,58 +47,91 @@ A quick way to start is by adding your App ID to the `application` section of th
 
 And that's it! Your app is now ready to use TelemetryDeck. Hit the build button to check if everything is working – if not, check out the troubleshooting section below.
 
-For more advanced configuration options, programmatic usage and information about signals, parameters and all other aspects of the SDK, check out the [README file](https://github.com/TelemetryDeck/KotlinSDK?tab=readme-ov-file#programmatic-usage).
-
 Feel free to browse the [TelemetryDeck SDK's source code](https://github.com/TelemetryDeck/KotlinSDK) as well. It's tiny and you'll see for yourself how TelemetryDeck is hashing user identifiers before they ever reach the server. Privacy, yay!
 
-## You're all set!
+## Verify Your Setup
+
+Build and run your app to verify that TelemetryDeck is properly integrated. The SDK automatically tracks user activity when the app starts or returns from the background.
+
+{% notewarning "Test Mode for Development Signals" %}
+If you're running the app from your IDE or development environment, signals will be tagged as **Test Signals**. You'll see them show up in the TelemetryDeck Dashboard when the **Test Mode** toggle under the tab bar is turned on.
+{% endnotewarning %}
+
+Open the TelemetryDeck Dashboard, navigate to "Explore > Recent Signals" and make sure "Test Mode" is enabled. You should see automatic signals appear after launching your app.
+
+---
 
-By default, the TelemetryDeck SDK will automatically track user activity based on when the app starts, or when it returns from the background. This means, that this is all you need to do to start sending signals to TelemetryDeck.
+{% noteinfo "Ready for Basic Insights" %}
+Congratulations! With just the SDK integration you've completed, TelemetryDeck will automatically track user sessions, app launches, and device information. This basic setup provides valuable built-in insights without any additional code.
 
-You can now go to the [Dashboard](https://dashboard.telemetrydeck.com/) and watch your signals pour in, for example in the "Recent Signals" list. Signals produced while working in your IDE will automatically be tagged as **Test Signals** so remember to enable Test Mode in the Dashboard to see your testing data your Insights and charts. Disable Test Mode in the dashboard to see your production data once your app is live.
+You can now build and release your app. Once users start using it, your TelemetryDeck dashboard will begin showing data about user behavior, device types, and other key metrics.
+{% endnoteinfo %}
 
 ## Privacy Policy and Opt-Out
```

Comment:
I think "Opt-Out" is a remnant of an old version (what are we opting out from?) and can be dropped.

Is the goal that this is a more detailed explainer of what is already covered by `privacy-faq` or vis-versa? One of the two should be simplified (for all SDKs).

Reply:
I removed the suffix " and Opt-Out" in the title! 👍